lib/pagerduty.rb

```diff
@@ -34,7 +42,74 @@ def get_oncall_user(params = {})
     data = get_resources(:oncalls, params)
     raise Exceptions::NoOncallUser if data.empty?
 
-    data.first.fetch(:user)
+    data[0][:user]
+  end
+
+  def get_user(id = '404stub')
```

Comment:
Is it possible to guard this method against nil id's without sending junk to pagerduty's api?

Maybe something like:
```
def get_user(id)
  raise(ArgumentError, 'Id not provided') unless id
  
  response = http.get "/users/#{id}"
  ...
end

Reply:
Good idea. I'll do that.

Reply:
Nice. I've done that, and the one I copied from.